core/src/test/resources/spaces/Hacking.stat

```diff
@@ -1,5 +1,4 @@
-maxColumn = 40
-<<< SKIP Spaces in paretheses
-function(a, b, c)
+<<<SKIP Spaces in after close braces
```

Comment:
so Hacking.stat looks like:
`maxColumn = 40
<<< SKIP Spaces in paretheses
function(a, b, c)
>>>
function( a, b, c )
<<<SKIP Spaces in after close braces
class Foo @Inject()(conf: Configuration)
>>>
class Foo @Inject() (conf: Configuration) `